rpcs/src/client/lightclient_impl.rs

```diff
@@ -36,18 +36,27 @@ impl RpcClientT for LightClientRpc {
 
             let id = Some(sub.id().to_owned());
             let stream = sub
-                .map_err(|e| RpcError::ClientError(Box::new(e)))
+                .map_err(|e| Error::Client(Box::new(e)))
                 .boxed();
 
             Ok(RawRpcSubscription { id, stream })
         })
     }
 }
 
-fn lc_err_to_rpc_err(err: LightClientRpcError) -> RpcError {
+fn lc_err_to_rpc_err(err: LightClientRpcError) -> Error {
```

Comment:
can we make this an into impl instead? Or stopped by the orphan-rule?

Reply:
I think we could yeah, but it would make the conversion more "public" I guess, and here I was thinking of it as an implementation detail!

Reply:
Done! Made the code a bit nicer in the end :)